locales/en.json

```diff
@@ -0,0 +1,126 @@
+{
+	"language_name": "English",
```

Comment:
My thought was that we only use it for crediting translators and otherwise the code should just display `language_name`